soc/nordic/nrf54h/ironside_se/include/nrf/ironside_se_boot_report.h

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_SOC_NORDIC_NRF54H_IRONSIDE_SE_BOOT_REPORT_INCLUDE_NRF_IRONSIDE_SE_BOOT_REPORT_H_
+#define ZEPHYR_SOC_NORDIC_NRF54H_IRONSIDE_SE_BOOT_REPORT_INCLUDE_NRF_IRONSIDE_SE_BOOT_REPORT_H_
+
+#include <stdint.h>
+#include <stddef.h>
+
+#define IRONSIDE_SE_BOOT_REPORT_LOCAL_DOMAIN_CONTEXT_SIZE                                          \
+	CONFIG_SOC_NRF54H20_IRONSIDE_SE_BOOT_REPORT_LOCAL_DOMAIN_CONTEXT_SIZE
+#define IRONSIDE_SE_BOOT_REPORT_RANDOM_DATA_SIZE                                                   \
+	CONFIG_SOC_NRF54H20_IRONSIDE_SE_BOOT_REPORT_RANDOM_DATA_SIZE
+
+/** @brief UICR error description contained in the boot report. */
+struct ironside_se_boot_report_uicr_error {
+	/** The type of error. A value of 0 indicates no error */
+	uint32_t error_type;
+	/** Error descriptions specific to each type of UICR error */
+	union {
+		/** RFU */
+		struct {
+			uint32_t rfu[4];
+		} rfu;
+	} description;
+};
+
+/** @brief IRONside boot report. */
+struct ironside_se_boot_report {
+	/** Magic value used to identify valid boot report */
+	uint32_t magic;
+	/** Firmware version of IRONside SE. 8bit MAJOR.MINOR.PATCH.TWEAK */
```

Comment:
We don't have a TWEAK version, we have SEQNUM. So I'd suggest something like
```suggestion
	/** Firmware version of IRONside SE. 8bit MAJOR.MINOR.PATCH.SEQNUM */
```